site/_posts/2017-11-21-im-all-about-the-bugs.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Last Week in ManageIQ: Because you know I'm all about the bugs"
+author: juliancheal
+date: 2017-11-21
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+It's that time of year again what with; Thanksgiving, Black Friday deals, and many other seasonal holidays around the corner. On everyone's mind though before we get to all that are… …bugs. Lots and lots of bugs. Well more specifically bug fixes. With the Gaprindashvili release just around the corner. The team is busy busy, fixing bugs and making the bestest release of ManageIQ yet.
+
+This issue of Last Week in ManageIQ is brought to you this week by [Julian](https://juliancheal.co.uk). And along for the ride this week are [37 pull requests merged][manageiq PRs merged] in ManageIQ core [45 pull requests merged][manageiq-ui-classic PRs merged] in the UI-Classic, and many more in our plethora of repositories.
```

Comment:
:P `The owner of juliancheal.co.uk has configured their website improperly`

Reply:
Welp. I'll just un-`https` the link for now.